pkgs/applications/misc/bottles/default.nix

```diff
@@ -95,6 +95,11 @@ python3Packages.buildPythonApplication rec {
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
   '';
 
+  passthru.updateScript = gitUpdater {
+    inherit pname version;
+    url = "https://github.com/bottlesdevs/Bottles.git";
```

Comment:
```suggestion
```

This branch is 20000+ commits behind master, please do a rebase since it does not contain #169999.

Reply:
Sorry, forget to push